### PR TITLE
Adding CREATE_ADMIN_USER=false to .env.sample

### PR DESCRIPTION
**Pull request by andrewoh531**
Will be needed locally when migration scripts are run (e.g. running tests)

### CODE REVIEW

File: api/.env.sample [hunk 1]
```diff
@@ -27,4 +27,6 @@ TWILIO_ACCOUNT_SID=
 TWILIO_AUTH_TOKEN=
 TWILIO_FROM_NUMBER=
 
+CREATE_ADMIN_USER=false
```
Comment (jonathaningram) [inline api/.env.sample]: Why is this needed locally? Won't the migration just gracefully exit if the env var is not set?

https://github.com/govau/notify/blob/3e886de5e8943418602ef751a4c2dc286bc5a8ff/api/migrations/versions/0200_create_admin_user.py#L24

Comment (andrewoh531) [reply api/.env.sample]: Actually you're right...too early in the week
